actors/miner/tests/util.rs

```diff
@@ -481,12 +483,22 @@ impl ActorHarness {
             );
         }
 
-        let result = rt
-            .call::<Actor>(
-                Method::PreCommitSectorBatch as u64,
-                &RawBytes::serialize(params.clone()).unwrap(),
-            )
-            .unwrap();
+        let result = rt.call::<Actor>(
+            Method::PreCommitSectorBatch as u64,
+            &RawBytes::serialize(params.clone()).unwrap(),
+        );
```

Comment:
Call rt.verify() before returning.  Some background: the verify call makes sure all the expected things and non of the unexpected things happened in the call and cleans the slate.  So any helper method that you calls actor code and then is called from tests should call it. Alternatively you could remember to call rt.verify after every call to this method but that's harder to remember.

Reply:
You can then remove rt.verify from the `pre_commit_sector_batch_and_get` method.  It should just be redundant however so it shouldnt' cause anything to break if you keep it.

Reply:
Make sure you're also doing this for your refactor of regular PreCommit

Reply:
I have problems making those changes as it breaks most of existing tests.

Reply:
My first thought is that this is catching legitimate failures in existing tests

Reply:
Something quite bad and unexpected is going on here.  I'm looking into it and will let you know when I've found out what's happening

Reply:
Ok I stand corrected.  The problem is that when this harness is used in tests that exercise failures the mock runtime expectations of caller validation and network info query are not always met because many checks happen and fail the call before the actor code gets here.  Leaving things the way they are is fine as long as we add in the rt.reset() call to `assert_simple_batch` so that it may be used without causing unexpected failures by future callers.